src/main/webapp/WEB-INF/tags/instructor/results/expandCollapseButton.tag

```diff
@@ -0,0 +1,21 @@
+<%@ tag description="instructorFeedbackResultsTop - Expand Collapse Button" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ attribute name="sortType" required="true" %>
+<%@ attribute name="showAll" required="true" %>
+<div class="pull-right" style="margin-top:7px">
```

Comment:
@whipermr5 alright. Because the div already had a pull right, after adding the margin-top of 7px, the expand collapse button shifts a bit closer to the question panel, without actually pushing the entire layout down. (eg https://user-images.githubusercontent.com/10449636/27985567-a05f9e00-640a-11e7-9dc0-b9b84653eddb.JPG) 

Having removed the pull-right and added a bottom margin, a similar effect can be achieved: 
![image](https://user-images.githubusercontent.com/10449636/28557529-f556d2ca-70da-11e7-86b7-df99e972da2b.png)

Please check the image at the bottom for how the UI would appear with the pull-right class removed. Is this acceptable ?

Additionally:
After removing the pull right, the margin-top is of no use, since it effectively changes nothing. Should I go ahead an remove the top margin as well ?

![image](https://user-images.githubusercontent.com/10449636/28557594-58be6ef4-70db-11e7-9250-76812628e333.png)

**Please let me know the design preferences.**